miden/tests/integration/operations/exec_iters.rs

```diff
@@ -0,0 +1,103 @@
+use super::super::build_test;
+use processor::VmState;
+use vm_core::Felt;
+
+// EXEC ITER TESTS
+// =================================================================
+#[test]
```

Comment:
This module should probably go one level higher (it is not testing individual VM operations).

Reply:
fixed